src/System.Diagnostics.Debug/tests/DebuggerTests.cs

```diff
@@ -11,19 +11,35 @@ public class DebuggerTests
         [Fact]
         public void IsAttached()
         {
-            // TODO: Implement this when Debugger is properly implemented
+            bool b = Debugger.IsAttached;
         }
 
         [Fact]
-        public void Launch()
+        public void Break()
         {
-            // TODO: Implement this when Debugger is properly implemented
+            if (!Debugger.IsAttached)
+                Debugger.Break();
```

Comment:
But calling Debugger.Break() when no debugger is attached will mess things up, at least when there's a real implementation of Debugger.Break() in use, i.e. this will fail on desktop.